src/main/flight/gps_rescue.h

```diff
@@ -45,13 +39,10 @@ typedef enum {
     GPS_RESCUE_ALT_MODE_COUNT
 } gpsRescueAltitudeMode_e;
 
-extern float gpsRescueAngle[RP_AXIS_COUNT]; // NOTE: ANGLES ARE IN CENTIDEGREES
-
 void gpsRescueInit(void);
 void gpsRescueUpdate(void);
 float gpsRescueGetYawRate(void);
 bool gpsRescueIsConfigured(void);
 bool gpsRescueIsAvailable(void);
-bool gpsRescueIsDisabled(void);
-bool gpsRescueDisableMag(void);
-float gpsRescueGetImuYawCogGain(void);
+bool gpsRescueIsOK(void);
+bool gpsRescueDisableMag(void);
```

Comment:
No extra line on EOF
```suggestion
bool gpsRescueDisableMag(void);

```

Reply:
thanks